packages/app-accounts/src/Creator.tsx

```diff
@@ -71,7 +71,7 @@ function isHexSeed (seed: string): boolean {
 }
 
 function rawValidate (seed: string): boolean {
-  return seed.length <= 32 || isHexSeed(seed);
+  return seed.length >= 32 || isHexSeed(seed);
```

Comment:
Finally played around a bit, think this should rather be `return seed.length > 0 && (seed.length <= 32 || isHexSeed(seed));`